src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -9218,6 +9267,78 @@ export class StudyViewPageStore
             return isMixedReferenceGenome(this.queriedPhysicalStudies.result);
         }
     }
+    private _clinicalEventFilters = observable.array<DataFilter>([], {
```

Comment:
from what i can see, we are never editing this array except by replacing all of its contents entirely.  this means that we could use a simple array assigned to an observable reference instead of observable.array, which is only necessary when you need to respond to mutations of the array itself.    what i mean by this is, 
```
@observable.ref 
private _clinicalEventsFilters = [];

```